src/screens/LinkProject/LinkProject.tsx

```diff
@@ -20,7 +20,7 @@ const ProjectQuery = gql`
   }
 `;
 
-export const SelectProject = () => {
+export const LinkProject = () => {
   const [{ data, fetching, error }] = useQuery({
```

Comment:
I feel like we should get the types working sooner rather than later or we'll write a bunch of code we need to fix later on.

Reply:
I did that yesterday, just hadn't committed it yet.

It's a bit tricky where to cut this PR so it can be reviewed, and start a new one, because I keep iterating on it.

Reply:
Oh actually I hadn't for LinkProject, because I'm not working on that in this PR. It's just a placeholder right now.